server/src/main/java/io/spine/server/aggregate/AggregateRepository.java

```diff
@@ -432,10 +432,14 @@ A loadOrCreate(I id) {
      * a {@linkplain #getSnapshotTrigger() snapshot trigger} is used as a
      * {@linkplain AggregateReadRequest#getBatchSize() batch size}.
      *
+     * <p>This method is exposed into the {@code io.spine.server.aggregate} package for the only
+     * reason of making it overridable. Consider calling {@link #loadOrCreate loadOrCreate()} or
+     * {@link #find find()} instead.
+     *
      * @param id the ID of the aggregate
      * @return the loaded instance or {@code Optional.absent()} if there is no record with the ID
      */
-    private Optional<A> load(I id) {
+    Optional<A> load(I id) {
```

Comment:
Considering the implementation details, I wouldn't make this method available even to its package. Let's discuss.